back/src/resolvers/mutation/createTimeTraveller.ts

```diff
@@ -0,0 +1,32 @@
+import { repos } from "../../db/dbconfig"
+
+interface CreateTimeTravellerInputModel {
+  name: string
+  birth: string
+  passport: number
+}
+
+export default {
+  createTimeTraveller: async (
```

Comment:
ai esse export poderia ser apenas
```suggestion
export const createTimeTravellerUseCase = async (
```

Reply:
Alterado export default de objeto para export de const nomeada com "UseCase" ao final.